src/p2p/net_node.inl

```diff
@@ -633,6 +635,115 @@ namespace nodetool
   }
   //-----------------------------------------------------------------------------------
   template<class t_payload_net_handler>
+  std::set<std::string> node_server<t_payload_net_handler>::get_seed_nodes()
+  {
+    if (m_nettype == cryptonote::TESTNET)
+    {
+      return get_seed_nodes(cryptonote::TESTNET);
+    }
+    if (m_nettype == cryptonote::STAGENET)
+    {
+      return get_seed_nodes(cryptonote::STAGENET);
+    }
+    if (!m_exclusive_peers.empty() || m_offline)
```

Comment:
Should this not be first ?

Reply:
Is the way it is done in the current code.

Yeah, moved it.